capsules/src/virtual_alarm.rs

```diff
@@ -4,7 +4,7 @@
 use core::cell::Cell;
 use kernel::common::cells::OptionalCell;
 use kernel::common::{List, ListLink, ListNode};
-use kernel::debug;
+//use kernel::debug;
```

Comment:
There are lots of debug prints that should be removed.

Reply:
Later.

Reply:
Isn't this a PR for ready for review and merge?